### ISSUE TITLE
testrun on Target x86 linux os

### ISSUE THREAD

**Issue by kush930** [google/testrun#325]
Hi all 

`I have cloned testrun from link`:- https://github.com/google/testrun.git  and `successfully installed testrun with all dependencies on my target machine x86 linux os ,as per given information from this link`- https://github.com/google/testrun/blob/main/docs/get_started.md

![testrun](https://github.com/google/testrun/assets/80834291/9168ab1f-deba-446c-8200-92264ff52b97)

`After completing all steps- I am getting issue of` **Testrun configuration has not yet been completed** in the bottom right corner of my attached image.

**Please let me know what have I missed here, any help is appreciated.**

T&R
kush

**Comment by jboddey** [comment]
Hi there, could you provide a screenshot of the system configuration available by clicking the settings icon in the top right? And also the file that is available at /usr/local/testrun/local/system.json?

**Comment by kush930** [comment]
Hi jboddey, 

**Thanks for reply, I am attaching here** `/usr/local/testrun/local/system.json`

```
![Screenshot from 2024-03-13 21-19-52](https://github.com/google/testrun/assets/80834291/1c15aa19-3d43-4fbc-b2b3-8d825eb04202)
![Screenshot from 2024-03-13 21-19-44](https://github.com/google/testrun/assets/80834291/f6ae4bd5-8b0e-45ae-bbc3-0266af5fbdde)


  "network": {
    "device_intf": "enx123456789123",
    "internet_intf": "enx123456789124"
  },
  "log_level": "INFO",
  "startup_timeout": 60,
  "monitor_period": 300,
  "max_device_reports": 5
}

```
T&R
Kush

**Comment by jboddey** [comment]
Hi, could you send a picture of the user interface when the settings menu is open? There is a button in the very top right corner. Could you press that and send a screenshot of the full menu?

**Comment by kush930** [comment]
Hi jboddey,

**I was doing some wrong steps, As per your guidance , I again started configuration , here one thing is making me confuse in setting  that How to use** `"Select port used for Internet connection"`,  **Another setting** `"Select port where you’ll connect the tested device"` **is taking automatically.**
![Screenshot from 2024-03-18 11-41-40](https://github.com/google/testrun/assets/80834291/2d944e24-60b2-4250-979d-5ef1a43b20fe)

**Can You guide me, How to do proper setting.**

T&R
Kush Kumar

**Comment by jboddey** [comment]
Hi Kush,

If you only have one port available, you can run Testrun using single interface mode as follows: `sudo testrun --single-intf`. You would then choose your Ethernet port as the Device connection port in the settings menu. Note you would then have to connect the ethernet port to your device and not your router. This method is discouraged as it will not provide the device with an internet connection during testing. Instead, you should acquire a USB ethernet port to add a second connection for your device.

**Comment by kush930** [comment]
Hi Jboddey,

**I went through** `your given instruction`, **I run the command** `sudo testrun --single-intf` **on my terminal , After setting what i am getting , that i am** `sharing with you in attachment`.

`Device is connected from my PC with using USB and Ethernet is connected from router and My PC also connected from Ethernet.  `

![Screenshot from 2024-03-18 18-03-09](https://github.com/google/testrun/assets/80834291/b39bb149-c4f6-4393-ba85-e42e2b94493a)

![Screenshot from 2024-03-18 18-02-48](https://github.com/google/testrun/assets/80834291/9c60aa4e-df53-4456-93db-a8254f4e3a88)

![connection](https://github.com/google/testrun/assets/80834291/ba262208-aecd-4387-93cc-18e779480fc0)

**Can you suggest me here, where I am doing mistake.**

T&R
Kush Kumar

**Comment by kush930** [comment]
Hi jboddey,

**As per your suggestion,** I executed the below command on my pc, **same issue I am facing**-
```
sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0
sudo sysctl -w net.ipv6.conf.default.disable_ipv6=0
sudo sysctl -w net.ipv6.conf.lo.disable_ipv6=0
```

```
After running testrun command on terminal, ethernet is not working, after killed testrun pid from terminal, same  ethrrnet issue , am facing, After restart system also same issue ethernet connection is coming.

Another machine I used, same issue I am getting.

```
Attaching here some screenshot

![MicrosoftTeams-image (2)](https://github.com/google/testrun/assets/80834291/662bbdf7-8a6b-4baf-9d5d-e58624187d3d)

![MicrosoftTeams-image (3)](https://github.com/google/testrun/assets/80834291/8a758313-6cad-4584-8fb7-9ad646417d75)

![MicrosoftTeams-image (4)](https://github.com/google/testrun/assets/80834291/004ff15f-7338-49a7-808f-004401904f05)

![MicrosoftTeams-image (5)](https://github.com/google/testrun/assets/80834291/adceadaf-ca07-44dc-a7fe-540e8dc3f8ba)


**Can you please guide me, where i am doing mistake.**

T&R
KUSH KUMAR

**Comment by jboddey** [comment]
Hi @kush930. How come you are pinging the 192.168.1.59 address? The interface used to connect to the device should be disabled in network settings in Ubuntu. The device must act as a DHCP client, and Testrun will act as the DHCP server.

**Comment by kush930** [comment]
Hi jboddey, 

`
Thanks, As per your suggestion I executed command, it's working fine now`. 

`could u please tell me , do i need to paste the .crt file in the root folder`. 

T&R
Kush Kumar

**Comment by jboddey** [comment]
The .crt file should be pasted into the /usr/local/testrun/local/root_certs folder.

**Comment by kush930** [comment]
Hello jboddey,
I have two devices, the reason for the failure of testrun  is attached below
![image](https://github.com/google/testrun/assets/80834291/681fa8bf-9b5d-437a-bf2b-44866d69c941)
![image](https://github.com/google/testrun/assets/80834291/18a903fa-789e-4960-87cc-ae48c4417fb4)

for second device the reason for non compliant is attched below
![image](https://github.com/google/testrun/assets/80834291/6192872a-49d0-412c-af86-d49510270240)

Could you please tell me how can I resolve this or make change so i can make my device compliant. It would be really helpful if you can give details of each non compliant parameter

NOTE : I had run these tests yesterday without pasting the .crt file in the root_certs folder.

**Comment by jboddey** [comment]
Hi @kush930, could you please email anniappan@google.com with information about your device and attach the ZIP file? He may then schedule a call between yourself and one of the testing labs.

**Comment by kush930** [comment]
Sure jboddey.  Please tell me what to attach in the zip file. Thanks for your help. :)

**Comment by kush930** [comment]
Hello jboddey, haven't received any reply from anniappan, need to resolve this issue quickly, could you pleas help me with that

**Comment by jboddey** [comment]
Sure, could you resend the email and cc me in: boddey@google.com

**Comment by kush930** [comment]
I have added you to the mail chain

**Comment by kush930** [comment]
Hi jboddey, 

After executing, `sudo testrun --single-intf ` on `linux`.  and started testrun on http://localhost:8080

Attaching here log message below-

**kush@enlite:~/Desktop$ sudo testrun --single-intf**

Apr 01 12:44:18 session  INFO    Running Testrun version 1.1.3
Apr 01 12:44:18 test_run INFO    Loading reports from /usr/local/testrun/local/devices/thermo thermo_1/reports
Apr 01 12:44:18 test_run INFO    Stopping user interface
Apr 01 12:44:18 test_run INFO    Starting UI
Apr 01 12:44:20 test_run INFO    User interface is ready on http://localhost:8080
Apr 01 12:44:20 api      INFO    Starting API
Apr 01 12:44:20 api      INFO    API waiting for requests
Apr 01 12:45:18 api      INFO    Starting Testrun with device target thermo thermo_1 with MAC address 08:3a:8d:29:ea:d8
Apr 01 12:45:18 net_orc  INFO    Loaded the following network modules: dns ntp template radius dhcp-2 gateway dhcp-1 
Apr 01 12:45:18 net_orc  INFO    Stopping network services
Apr 01 12:45:18 net_orc  INFO    Clearing baseline network
Apr 01 12:45:18 net_orc  INFO    Network is restored
Apr 01 12:45:18 net_orc  INFO    Starting network
Apr 01 12:45:18 net_orc  INFO    Creating baseline network
Apr 01 12:45:18 net_orc  INFO    Starting network services
```
Apr 01 12:45:21 util     ERROR   Command failed: ip netns exec tr-ctns-dns ip addr add fd10:77be:4186::4/64 dev veth0
Apr 01 12:45:21 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
Apr 01 12:45:24 util     ERROR   Command failed: ip netns exec tr-ctns-ntp ip addr add fd10:77be:4186::5/64 dev veth0
Apr 01 12:45:24 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
Apr 01 12:45:28 util     ERROR   Command failed: ip netns exec tr-ctns-radius ip addr add fd10:77be:4186::7/64 dev veth0
Apr 01 12:45:28 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
Apr 01 12:45:31 util     ERROR   Command failed: ip netns exec tr-ctns-dhcp-2 ip addr add fd10:77be:4186::3/64 dev veth0
Apr 01 12:45:31 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
Apr 01 12:45:34 util     ERROR   Command failed: ip netns exec tr-ctns-gateway ip addr add fd10:77be:4186::1/64 dev veth0
Apr 01 12:45:34 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
Apr 01 12:45:38 util     ERROR   Command failed: ip netns exec tr-ctns-dhcp-1 ip addr add fd10:77be:4186::2/64 dev veth0
Apr 01 12:45:38 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
```
Apr 01 12:45:38 net_orc  INFO    All network services are running
Apr 01 12:45:38 test_orc INFO    Loaded the following test modules: protocol dns ntp tls baseline nmap conn 
Apr 01 12:45:38 test_run INFO    Waiting for devices on the network...
Apr 01 12:45:43 test_run INFO    Discovered thermo thermo_1 on the network. Waiting for device to obtain IP
Apr 01 12:46:00 net_orc  INFO    Device with mac addr 08:3a:8d:29:ea:d8 has obtained IP address 10.10.10.14
Apr 01 12:46:00 net_orc  INFO    Monitoring device with mac addr 08:3a:8d:29:ea:d8 for 300 seconds
Apr 01 12:51:00 test_run INFO    Device with mac address 08:3a:8d:29:ea:d8 is ready for testing.
Apr 01 12:51:00 test_orc INFO    Running test modules on device with mac addr 08:3a:8d:29:ea:d8
Apr 01 12:51:00 test_orc INFO    Running test module dns
Apr 01 12:51:08 test_dns INFO    Running dns.network.hostname_resolution
Apr 01 12:51:08 test_dns INFO    Checking DNS traffic from device: 08:3a:8d:29:ea:d8
Apr 01 12:51:08 test_dns INFO    DNS Server queries found: 1
Apr 01 12:51:08 test_dns INFO    Startup DNS queries found: 0
Apr 01 12:51:08 test_dns INFO    Monitor DNS queries found: 1
Apr 01 12:51:08 test_dns INFO    DNS queries found: 2
Apr 01 12:51:08 test_dns INFO    DNS traffic detected from device
Apr 01 12:51:08 test_dns INFO    Running dns.network.from_dhcp
Apr 01 12:51:08 test_dns INFO    Checking DNS traffic for configured DHCP DNS server: 10.10.10.4
Apr 01 12:51:08 test_dns INFO    DNS Server queries found: 1
Apr 01 12:51:08 test_dns INFO    Startup DNS queries found: 0
Apr 01 12:51:08 test_dns INFO    Monitor DNS queries found: 1
Apr 01 12:51:08 test_dns INFO    DNS queries found: 2
Apr 01 12:51:08 test_dns INFO    DNS Server queries found: 0
Apr 01 12:51:08 test_dns INFO    Startup DNS queries found: 0
Apr 01 12:51:08 test_dns INFO    Monitor DNS queries found: 0
Apr 01 12:51:08 test_dns INFO    DNS queries found: 0
Apr 01 12:51:08 test_dns INFO    DNS traffic detected only to configured DHCP DNS server
Apr 01 12:51:08 test_dns INFO    Writing results to /runtime/output/dns-result.json
Apr 01 12:51:10 test_orc INFO    Test module dns has finished
Apr 01 12:51:10 test_orc INFO    Running test module ntp
Apr 01 12:51:18 test_ntp INFO    Running ntp.network.ntp_support
Apr 01 12:51:18 test_ntp INFO    Device sent TLSNTPv4 request to 216.239.35.4
Apr 01 12:51:18 test_ntp INFO    Device sent NTPv4 request to 216.239.35.4
Apr 01 12:51:18 test_ntp INFO    Device sent NTPv4 request to 216.239.35.4
Apr 01 12:51:18 test_ntp INFO    Device sent NTPv4 request to 216.239.35.4
Apr 01 12:51:18 test_ntp INFO    Device sent NTPv4 packets.
Apr 01 12:51:18 test_ntp INFO    Running ntp.network.ntp_dhcp
Apr 01 12:51:18 test_ntp INFO    Device sent NTP request to non-DHCP provided NTP server
Apr 01 12:51:18 test_ntp INFO    Device sent NTP request to non-DHCP provided NTP server
Apr 01 12:51:18 test_ntp INFO    Device sent NTP request to non-DHCP provided NTP server
Apr 01 12:51:18 test_ntp INFO    Device sent NTP request to non-DHCP provided NTP server
Apr 01 12:51:18 test_ntp INFO    Device sent NTP request to non-DHCP provided server
Apr 01 12:51:18 test_ntp INFO    Writing results to /runtime/output/ntp-result.json
Apr 01 12:51:20 test_orc INFO    Test module ntp has finished
Apr 01 12:51:20 test_orc INFO    Running test module tls
```
Apr 01 12:51:23 util     ERROR   Command failed: ip netns exec tr-test-tls ip addr add fd10:77be:4186::9/64 dev veth0
Apr 01 12:51:23 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
```
Apr 01 12:51:29 test_module INFO    Starting TLS Module
Apr 01 12:51:31 test_tls INFO    Device IP Resolved: 10.10.10.14
Apr 01 12:51:31 test_tls INFO    Running security.tls.v1_2_server
Apr 01 12:51:31 test_tls INFO    Connection to 10.10.10.14:443 was refused.
Apr 01 12:51:31 test_tls INFO    Failed to resolve public certificate
Apr 01 12:51:31 test_tls INFO    Connection to 10.10.10.14:443 was refused.
Apr 01 12:51:31 test_tls INFO    Failed to resolve public certificate
Apr 01 12:51:31 test_tls INFO    TLS server test results: (None, 'TLS 1.2 not validated: Failed to resolve public certificate\nTLS 1.3 not validated: Failed to resolve public certificate')
Apr 01 12:51:31 test_tls INFO    Running security.tls.v1_2_client
Apr 01 12:51:31 test_tls INFO    Validating client for TLS: 1.2
Apr 01 12:51:31 test_tls INFO    No client hello packets detected
Apr 01 12:51:31 test_tls INFO    Checking client for non-TLS client connections
Apr 01 12:51:32 test_tls INFO    Checking client for TLS client connections
Apr 01 12:51:33 test_tls INFO    Checking client for unsupported TLS client connections
Apr 01 12:51:34 test_tls INFO    Writing results to /runtime/output/tls-result.json
Apr 01 12:51:36 test_orc INFO    Test module tls has finished
Apr 01 12:51:36 test_orc INFO    Running test module nmap
```
Apr 01 12:51:39 util     ERROR   Command failed: ip netns exec tr-test-nmap ip addr add fd10:77be:4186::9/64 dev veth0
Apr 01 12:51:39 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
```
Apr 01 12:51:44 test_nmap INFO    Running nmap module
Apr 01 12:51:44 test_nmap INFO    Running nmap TCP port scan
Apr 01 12:51:44 test_nmap INFO    Running nmap UDP port scan
Apr 01 12:51:47 test_nmap INFO    UDP port scan complete
Apr 01 12:52:01 test_nmap INFO    TCP port scan complete
Apr 01 12:52:07 test_nmap INFO    Device IP Resolved: 10.10.10.14
Apr 01 12:52:07 test_nmap INFO    Running security.services.ftp
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.ssh.version
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.telnet
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.smtp
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.http
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.pop
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.imap
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.snmpv3
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running ntp.services.vnc
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running security.services.tftp
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Running ntp.network.ntp_server
Apr 01 12:52:07 test_nmap INFO    Checking results
Apr 01 12:52:07 test_nmap INFO    Writing results to /runtime/output/nmap-result.json
Apr 01 12:52:09 test_orc INFO    Test module nmap has finished
Apr 01 12:52:09 test_orc INFO    Running test module connection
```
Apr 01 12:52:12 util     ERROR   Command failed: ip netns exec tr-test-conn ip addr add fd10:77be:4186::9/64 dev veth0
Apr 01 12:52:12 util     ERROR   Error: b'RTNETLINK answers: Permission denied'. Code: 2
```
Apr 01 12:52:17 test_connection INFO    Starting connection module
Apr 01 12:52:27 test_connection INFO    Device IP Resolved: 10.10.10.14
Apr 01 12:52:27 test_connection INFO    Running connection.dhcp_address
Apr 01 12:52:27 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:52:27 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:52:27 test_connection INFO    DHCP Lease resolved:
Apr 01 12:52:27 test_connection INFO    IP Resolved: 10.10.10.14
Apr 01 12:52:27 test_connection INFO    Attempting to ping device...
Apr 01 12:52:27 test_connection INFO    Pinging: 10.10.10.14
Apr 01 12:52:27 test_connection INFO    Running connection.mac_address
Apr 01 12:52:27 test_connection INFO    MAC address found: 08:3a:8d:29:ea:d8
Apr 01 12:52:27 test_connection INFO    Running connection.mac_oui
Apr 01 12:52:27 test_connection INFO    OUI Manufacturer found: Espressif Inc.
Apr 01 12:52:27 test_connection INFO    Running connection.private_address
Apr 01 12:52:27 test_connection INFO    Current DHCP subnet range: {'start': '10.10.10.10', 'end': '10.10.10.20'}
Apr 01 12:52:27 test_connection INFO    Stopping secondary DHCP server
Apr 01 12:52:27 test_connection INFO    Secondary DHCP server stop command success
Apr 01 12:52:30 test_connection INFO    Checking secondary DHCP server status
Apr 01 12:52:30 test_connection INFO    Secondary DHCP server stopped
Apr 01 12:52:30 test_connection INFO    Configuring primary DHCP server
Apr 01 12:52:30 test_connection INFO    Primary DHCP server failover disabled
Apr 01 12:52:30 test_connection INFO    Single DHCP server configured
Apr 01 12:52:30 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:52:30 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:52:30 test_connection INFO    DHCP Lease resolved:
Apr 01 12:52:30 test_connection INFO    Lease IP Resolved: 10.10.10.14
Apr 01 12:52:30 test_connection INFO    Attempting to ping device...
Apr 01 12:52:30 test_connection INFO    Ping Success: True
Apr 01 12:52:30 test_connection INFO    Current lease confirmed active in device
Apr 01 12:52:30 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:52:36 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:52:36 test_connection INFO    DHCP Lease resolved:
Apr 01 12:52:36 test_connection INFO    Testing subnet: {'start': '10.0.0.100', 'end': '10.0.0.200'}
Apr 01 12:52:36 test_connection INFO    Changing subnet to: {'start': '10.0.0.100', 'end': '10.0.0.200'}
Apr 01 12:52:36 test_connection INFO    Time until lease expiration: 15.991194
Apr 01 12:52:36 test_connection INFO    Waiting for current lease to expire: 2024-04-01 12:52:47+05:30
Apr 01 12:52:52 test_connection INFO    Current lease expired.
Apr 01 12:52:52 test_connection INFO    Resolving current lease with max wait time of 120 seconds
Apr 01 12:52:52 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:52:52 test_connection INFO    DHCP Lease resolved:
Apr 01 12:52:52 test_connection INFO    Lease within subnet: True
Apr 01 12:52:52 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:52:52 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:52:52 test_connection INFO    DHCP Lease resolved:
Apr 01 12:52:52 test_connection INFO    Testing subnet: {'start': '172.16.0.0', 'end': '172.16.255.255'}
Apr 01 12:52:52 test_connection INFO    Changing subnet to: {'start': '172.16.0.0', 'end': '172.16.255.255'}
Apr 01 12:52:52 test_connection INFO    Time until lease expiration: 30.943506
Apr 01 12:52:52 test_connection INFO    Waiting for current lease to expire: 2024-04-01 12:53:18+05:30
Apr 01 12:53:23 test_connection INFO    Current lease expired.
Apr 01 12:53:23 test_connection INFO    Resolving current lease with max wait time of 120 seconds
Apr 01 12:53:23 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:53:23 test_connection INFO    DHCP Lease resolved:
Apr 01 12:53:23 test_connection INFO    Lease within subnet: True
Apr 01 12:53:23 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:53:23 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:53:23 test_connection INFO    DHCP Lease resolved:
Apr 01 12:53:23 test_connection INFO    Testing subnet: {'start': '192.168.0.0', 'end': '192.168.255.255'}
Apr 01 12:53:23 test_connection INFO    Changing subnet to: {'start': '192.168.0.0', 'end': '192.168.255.255'}
Apr 01 12:53:23 test_connection INFO    Time until lease expiration: 31.922317
Apr 01 12:53:23 test_connection INFO    Waiting for current lease to expire: 2024-04-01 12:53:50+05:30
Apr 01 12:53:55 test_connection INFO    Current lease expired.
Apr 01 12:53:55 test_connection INFO    Resolving current lease with max wait time of 120 seconds
Apr 01 12:53:55 test_connection INFO    DHCP lease resolved from primary server
Apr 01 12:53:55 test_connection INFO    DHCP Lease resolved:
Apr 01 12:53:55 test_connection INFO    Lease within subnet: True
Apr 01 12:53:55 test_connection INFO    Changing subnet to: {'start': '10.10.10.10', 'end': '10.10.10.20'}
Apr 01 12:53:55 test_connection INFO    Configuring primary failover DHCP server
Apr 01 12:53:55 test_connection INFO    Primary DHCP server enabled
Apr 01 12:53:55 test_connection INFO    DHCP server configuration restored
Apr 01 12:53:55 test_connection INFO    Resolving current lease with max wait time of 60 seconds
Apr 01 12:54:25 test_connection INFO    DHCP lease resolved from secondary server
Apr 01 12:54:25 test_connection INFO    DHCP Lease resolved:
Apr 01 12:54:25 test_connection INFO    Time until lease expiration: 33.693965
Apr 01 12:54:25 test_connection INFO    Waiting for current lease to expire: 2024-04-01 12:54:54+05:30

Can anyone help me to resolve this issue.

T&R
Kush Kumar

**Comment by jboddey** [comment]
Closing this issue as the support request has now moved to email.
